packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -22,6 +22,28 @@ void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
   GoogleMapsFlutterPlatform.instance.enableDebugInspection();
 
+  // Repeatedly checks an asynchronous value against a test condition, waiting
+  // one frame between each check, returing the value if it passes the predicate
```

Comment:
Is one frame in this case based on the actual frame rate of the application? Or does it mean something else?

Reply:
It's just what `tester.pump()` does. It could just as easily be "wait some arbitrary short amount of time"; `pump` was just a convenient way to do that in widget tests.